application-helm/templates/create-mysql-app-db-user.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Values.initdb -}}
+{{- if .Values.initdb.mysql.auto_db_user_create -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: {{ .Values.environment }}-{{ template "application.name" $ }}-app-db-user-creation-
```

Comment:
wouldnt this be a very big name? I believe K8s has some limit on resources name, not sure what that is though?